python/rmm/rmm/_lib/memory_resource.pyx

```diff
@@ -366,6 +376,38 @@ cdef class ManagedMemoryResource(DeviceMemoryResource):
         pass
 
 
+cdef class SystemMemoryResource(DeviceMemoryResource):
```

Comment:
I think we should mirror the C++ objects here. If we don't want the C++ `system_memory_resource` to have a headroom parameter, we should make the same choice in Python.

Reply:
Done.